resources/guidelines/documentation-guidelines/07-embedding-external-repositories.md

```diff
@@ -0,0 +1,225 @@
+# Embedding external repositories
+
+In this guide, we will explain how to embed project documentation from your repository into the [Developer documentation](https://developer.shopware.com/frontends/).
+
+[Developer Portal](https://github.com/shopware/developer-portal) is built on top of [`shopware/developer-documentation-vitepress`](https://github.com/shopware/developer-documentation-vitepress) repository (`vitepress-shopware-docs` package) which heavily utilizes [Vitepress](https://vitepress.dev/) and customizes it with custom Shopware features such as custom design, breadcrumbs, Algolia search, Copilot AI chat and recommendations, auto-built sidebar and more.
+
+It acts as a central point for all developer resources and documentation. However, the actual content is stored in various repositories and embedded into the developer portal with help of the [Docs CLI](https://github.com/shopware/developer-documentation-vitepress/blob/main/CLI.md). This allows for a decentralized approach to managing the content and enables the maintainers of the respective repositories to manage their content independently.
+
+## Local instance of the Developer Portal
+
+To set up your local instance of the developer portal, clone Developer Portal repository and install the dependencies:
+
+```bash
+cd /www/shopware/
+git clone https://github.com/shopware/developer-portal.git
+cd developer-portal
+pnpm i
+```
+
+We also want to create a new branch.
+
+```bash
+git checkout -b feature/embed-meteor-icon-kit
+```
+
+### Docs CLI
+
+Docs CLI is now accessible by running `./docs-cli` in the root of the `shopware/developer-portal` repository.
+
+To start embedding a new repository, update `.vitepress/portal.json` and create a new entry in the `repositories` array. Then run the CLI and see if your repository is visible in the list - select it and continue by confirming the default settings.
+
+```bash
+./docs-cli manage
+```
+
+You should be able to preview your new content by running the Vitepress dev server and opening your defined URL in the browser.
+
+```bash
+pnpm dev
+```
+
+### Sidebar and main navigation
+
+The content is already there and published, but in most cases you'll also want to have a sidebar dedicated for your section.
+
+Open `.vitepress/navigation.ts` and update `sublinks` and `ignore` parameters to auto-build the sidebar based on your directory structure and frontmatter config.
+
+If you also want to add it to the top-bar main menu, update the `navigation` accordingly.
+
+### Algolia search
+
+By default, contents are grouped under `General` section in the Algolia search using Algolia _facets_. You can configure that and group your articles together into a new section, or even create multiple new sections.
```

Comment:
```suggestion
By default, contents are grouped under `General` section in the Algolia search using `Algolia _facets_`. You can configure that and group your articles together into a new section, or even create multiple new sections.
```

Reply:
Algolia is name of the company/product, and _facets_ is their term for tags or additional metadata. Because it's a term, it feels like it should stay _italic_.